pages/git-protocol.md

```diff
@@ -21,7 +23,9 @@ A guide for programming within version control.
 * Perform work in a feature branch.
 * Rebase frequently to incorporate upstream changes.
 * Use a [pull request] for code reviews.
+* [Sign your commits with a GPG key](https://help.github.com/categories/gpg/).
 
+[2FA]: https://help.github.com/articles/about-two-factor-authentication/
```

Comment:
Interesting. I thought this was a common shortcut. I usually use this syntax for long URLs to keep lines under 80 characters and more readable. What would you have done?